Assets/Scenes/Forrest/MagicTree.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(SpriteRenderer))]
+public class MagicTree : MonoBehaviour
```

Comment:
Perhaps you should a small documentation for this class?

Reply:
@delvh  I corrected the error.